privatemarketplace/quickstart/aspire/README.md

```diff
@@ -0,0 +1,379 @@
+# VS Code Private Marketplace - Quickstart
+
+This quickstart walks you through setting up and testing a local VS Code Private Marketplace using [Aspire](https://aspire.dev) on Windows. You'll learn how to install the marketplace, configure VS Code to use it, and explore different usage scenarios.
+
+---
+
+## Part 1: Installation
+
+### Prerequisites
+
+Before you begin, ensure you have:
+- **Docker Desktop** installed and running
+- **PowerShell 5.1 or later** for running the setup script
+
+### Run the Setup Script
+
+> [!IMPORTANT]
+  Never run scripts from untrusted sources, always review the script before running it.
+  Always verify the script's hash before executing. The expected hash can be found in the repository or release notes.
+  Open PowerShell and run the quickstart script:
+
+The script will automatically:
+- Check for and install missing prerequisites (after prompting for confirmation):
+  - Docker Desktop (if not found)
+  - Download quickstart files to `$env:TEMP\privatemarketplace-quickstart`
+  - Portable VS Code
+  - Portable .NET SDK 10.0.100
```

Comment:
Why pin 10.0.100?  In a few weeks 10.0.101 will be available, but this script will only install/recognize 10.0.100.  When .NET releases security updates to .NET 10.0.100, users won't get them.

The dotnet-install script can install the latest patch in the same band:

`dotnet-install.ps1 -Channel 10.0.1xx -Quality GA`

I think this would be more secure than pinning 10.0.100. It will also minimize the need to manually retest this script against future .NET releases (vs. installing newer feature bands automatically).